src/DistributedLock.Azure/BlobClientWrapper.cs

```diff
@@ -25,9 +25,10 @@ public BlobClientWrapper(BlobBaseClient blobClient)
     {
         var conditions = new BlobRequestConditions { LeaseId = leaseId };
         var properties = SyncViaAsync.IsSynchronous
-            ? this._blobClient.GetProperties(conditions, cancellationToken)
-            : await this._blobClient.GetPropertiesAsync(conditions, cancellationToken).ConfigureAwait(false);
-        return properties.Value.Metadata;
+            ? _blobClient.GetProperties(conditions, cancellationToken).AsTask()
+            : _blobClient.GetPropertiesAsync(conditions, cancellationToken);
```

Comment:
Please restore use of `this.`. I prefer to use it consistently and you'll see that all files in the solution do this.